internal/cmd/operator-sdk/generate/bundle/bundle.go

```diff
@@ -291,13 +291,15 @@ func (c bundleCmd) runMetadata() error {
 		}
 	}
 
+	scorecardConfigPath := filepath.Join(bundleRoot, scorecard.DefaultConfigDir, scorecard.ConfigFileName)
```

Comment:
missing changelog

Reply:
Yes, waiting until 1.21 PR gets merged and then will add it.

Reply:
Added changelog.